notebooks/text_models/solutions/load_text.ipynb

```diff
@@ -0,0 +1,1935 @@
+{
```

Comment:
<p>Maybe add </p><p> </p><pre class="ql-syntax" spellcheck="false">pip freeze | grep tensorflow_text || pip install "tensorflow-text==2.8.*"
</pre><p> </p><p>or add the install in the top-level <code>requirements.txt</code> file </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I'll add it to the requirements.txt to keep it similar to other labs.</p>